gst/tensor_mux/gsttensormux.c

```diff
@@ -358,17 +402,51 @@ gst_tensor_mux_collect_buffer (GstTensorMux * tensor_mux,
     walk = g_slist_next (walk);
 
     GstBuffer *buf = NULL;
-    buf = gst_collect_pads_pop (tensor_mux->collect, data);
+
+    if (tensor_mux->synch) {
+      buf = gst_collect_pads_peek (tensor_mux->collect, data);
+      if (buf == NULL && pad->buffer == NULL)
+        return TRUE;
+      if (buf != NULL) {
+        if (GST_BUFFER_PTS (buf) < tensor_mux->current_time) {
+          gst_buffer_unref (buf);
+          buf = gst_collect_pads_pop (tensor_mux->collect, data);
+          if (pad->buffer != NULL)
+            gst_buffer_unref (pad->buffer);
+          pad->buffer = buf;
+          silent_debug ("Fame Dropped (# %d) : %lu", counting,
+              GST_BUFFER_PTS (buf));
+          tensor_mux->need_buffer = TRUE;
+          return FALSE;
+        }
+        if (pad->buffer != NULL
+            && ABS (tensor_mux->current_time - GST_BUFFER_PTS (pad->buffer) <
```

Comment:
check condition
ABS (tensor_mux->current_time - GST_BUFFER_PTS (pad->buffer)) < ABS (tensor_mux->current_time - GST_BUFFER_PTS (buf)))

Reply:
To prevent coverity issue, change code time-diff with GST_CLOCK_DIFF.
( please check PR #726 and #728 )